cherrypy/lib/static.py

```diff
@@ -28,6 +29,21 @@ def _setup_mimetypes():
 _setup_mimetypes()
 
 
+def _utf8_content_disposition(disposition, file_name):
+    """Create HTTP header for downloading a file with UTF-8 filename.
+
+    See this and related answers:
+    https://stackoverflow.com/a/8996249/2173868
+    """
+    ascii_name = unicodedata.normalize(
+        'NFKD', file_name).encode('ascii', 'ignore').decode()
+    header = '{}; filename="{}"'.format(disposition, ascii_name)
+    if ascii_name != file_name:
```

Comment:
Should we make this conditional? Adding it unconditionally would simplify the code.

Reply:
We don`t have to, its ok to simplify it.

Reply:
We don`t have to, its ok to simplify it.